src/apps/chifra/pkg/blockRange/range.go

```diff
@@ -73,6 +74,14 @@ type BlockRange struct {
 	Modifier     Modifier
 }
 
+func (br BlockRange) String() string {
```

Comment:
This is actually producing a JSON string, right? Maybe we want to call this function `MarshallJSON` and keep `String` for a potential future use (e.g. at some point we may want to serialize BlockRange to `0-10000`)

Reply:
Changing now.

Reply:
Should it take a pointer to the object or the object itself?